app/assets/javascripts/discourse/app/components/modal/share-topic.js

```diff
@@ -112,11 +115,5 @@ const ShareTopicModal = Component.extend(bufferedProperty("invite"), {
     const topicController = getOwner(this).lookup("controller:topic");
     topicController.actions.replyAsNewTopic.call(topicController, post);
     this.closeModal();
-  },
-});
-
-ShareTopicModal.reopenClass({
-  modalClass: "share-topic-modal",
```

Comment:
so this wasn't used at all? 😃

Reply:
Not as far as I can see. It's a 'static property' - very strange.

I suspect (but haven't checked) it was used by the old modal system.

Reply:
Now, it's in the template

https://github.com/discourse/discourse/blob/70ea3f8fcd2959a1435fb100afd0e53872811829/app/assets/javascripts/discourse/app/components/modal/share-topic.hbs#L11C10-L11C10